packages/gatsby-codemods/README.md

```diff
@@ -116,6 +116,36 @@ export default props => (
 )
 ```
 
+#### `rename-bound-action-creators`
+
+Rename `boundActionCreators` to `actions`. `boundActionCreators` has been deprecated in Gatsby v2
+
+Note: Run this codemod only against files that use `boundActionCreators` instead of running it against a whole directory.
```

Comment:
@DSchau @pieh does this Note make sense?

Reply:
Yeah, I think that's good!